SQLite.CodeFirst/SqliteConnectionStringParser.cs

```diff
@@ -26,6 +28,11 @@ internal static class SqliteConnectionStringParser
 
         public static string GetDataSource(string connectionString)
         {
+            if (connectionString.ToLower().Contains(DataDirectoryVariable.ToLower()))
+            {
+                var baseDirectory = AppDomain.CurrentDomain.BaseDirectory + @"\";
```

Comment:
Unfortunately, this variable is not always set. Probably only in ASP.NET. Please try it...